src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -280,8 +294,109 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
     }
   }
 
+  /** Encapsulates a {@code goog.require(...)} statement. */
```

Comment:
'A single statement containing goog.require.' otherwise it is not clear that you cover 'const {a,b} = goog.require(...)'.

Reply:
Done.